cmd/trust-manager/app/options/options.go

```diff
@@ -187,6 +194,14 @@ func (o *Options) addAppFlags(fs *pflag.FlagSet) {
 		"readiness-probe-path", "/readyz",
 		"HTTP path to expose the readiness probe server.")
 
+	fs.DurationVar(o.LeaseDuration,
+		"lease-duration", time.Second*60,
```

Comment:
Let's start with making these durations configurable without changing the defaults.
In another PR, we can increase these values and discuss if that makes sense there.

Reply:
Ok will change that. Those defaults are really bad. Even very mainstream controller (i.e. nginx-ingress) increase this by default (nginx-ingress does 30s renew for example).